lib/sequent/support/database.rb

```diff
@@ -53,7 +53,6 @@ def self.establish_connection(db_config, db_config_overrides = {})
           db_config = db_config.deep_merge(
             Sequent.configuration.primary_database_key => db_config_overrides,
           ).stringify_keys
-          ActiveRecord.legacy_connection_handling = false
```

Comment:
We might need to keep this if AR_VERSION is < 7.2 ? @sam-cw ?

Reply:
Hmm yes, I got failing tests telling me to remove it.

Reply:
It's fixed.